.babelrc

```diff
@@ -1,5 +1,8 @@
 {
   "optional": [
     "es7.objectRestSpread"
+  ],
+  "plugins": [
+    "babel-plugin-object-assign"
```

Comment:
According to this documentation: https://www.npmjs.com/package/babel-plugin-object-assign

Can you just declare:

```
"plugins": [
  "object-assign"
]
```